diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -1431,3 +1436,30 @@ void DifferentialFunctionComparator::findDifference(
         }
     }
 }
+
+bool DifferentialFunctionComparator::equal(Instruction *InstL,
+                                           Instruction *InstR) {
+    const auto *ValL = dyn_cast<Value>(InstL);
+    const auto *ValR = dyn_cast<Value>(InstR);
+
+    if (!ValL || !ValR)
+        return false;
```

Comment:
This shouldn't be necessary (`Instruction` inherits from `Value` and so `Instruction *` can be used directly where `Value *` is expected).